tests/tendermint/test_event_stream.py

```diff
@@ -35,6 +35,8 @@ def test_process_event_new_block():
     event_queue = Queue()
     process_event(event_queue, event, 'test_stream_id')
     assert not event_queue.empty()
+    block = event_queue.get()
+    assert type(block.data['id']) == int
```

Comment:
From [the Python3 documentation about `type()`](https://docs.python.org/3/library/functions.html#type):

> The `isinstance()` built-in function is recommended for testing the type of an object, because it takes subclasses into account.

I'm not sure if that matters here.  I looked it up and, `int` _does_ have subclasses: apparently "[Booleans are a subtype of integers.](https://docs.python.org/3.6/library/stdtypes.html#numeric-types-int-float-complex)"

I'll let you decide what to use.